packages/apollo-server-hapi/src/__tests__/ApolloServer.test.ts

```diff
@@ -1,14 +1,5 @@
-const NODE_VERSION = process.version.split('.');
-const NODE_MAJOR_VERSION = parseInt(NODE_VERSION[0].replace(/^v/, ''));
-
-// Skip hapi tests for unsupported versions of node
-if (NODE_MAJOR_VERSION < 8) {
-  it('does not run for node versions < 8', () => {});
-  return;
-}
-
-import { Server } from 'hapi';
 import {
+  atLeastMajorNodeVersion,
```

Comment:
This should be an easier to find helper method that we can use in the future when it comes time to shift Node.js version support.